### ISSUE TITLE
Emitters power adjacent containment field generators.

### ISSUE THREAD

**Issue by Simyon264** [space-wizards/space-station-14#25528]
## Description
Emitters power adjacent containment field generators. This feels like it's not intended.
**Reproduction & Screenshots**
![image](https://github.com/space-wizards/space-station-14/assets/63975668/7a9f82a9-52c2-4239-b31b-cd4ca26cf7ef)

**Comment by MACMAN2003** [comment]
i believe it is intended functionality

**Comment by Simyon264** [comment]
It feels wrong. It makes constructing containment fields for teslas too easy.

**Comment by PJB3005** [comment]
They are in fact supposed to share energy, and I'm not sure it's wrong that you should be able to charge a 3x3 field like the image with a single emitter, power usage should probably be in function of total field size.

That said, having the emitter pointed at the field like in the image is wrong.

I don't know about Tesla balance but the Tesla in general seems like a massive game design L so fix the Tesla instead.

**Comment by deltanedas** [comment]
yeah they are meant to share between cfgs but not just hitting a field

**Comment by Errant-4** [comment]
But you would have had to start all the generators up _already_ to be able to do this, otherwise there is no field to hit

**Comment by Simyon264** [comment]
> But you would have had to start all the generators up _already_ to be able to do this, otherwise there is no field to hit

No. You only need one emitter to power 4 generators.
